patterns/nvidia-gpu-efa/generate-efa-nccl-test.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+export MPI_JOB_NAME=efa-nccl-test
+export IMAGE_URI=public.ecr.aws/hpc-cloud/nccl-tests:latest
+export INSTANCE_TYPE=p5.48xlarge
+export NUM_WORKERS=2
+export GPU_PER_WORKER=8
+export EFA_PER_WORKER=32
+export TOTAL_GPUS=$((${NUM_WORKERS}*${GPU_PER_WORKER}))
+
+export NCCL_NVLS_ENABLE=1
```

Comment:
most of these can/should/must be removed https://github.com/aws/aws-ofi-nccl/blob/master/doc/efa-env-var.md

Reply:
Here is a ref for env parameter for NCCL, https://github.com/aws-samples/awsome-distributed-training/blob/main/micro-benchmarks/nccl-tests/kubernetes/nccl-tests.yaml#L40-L50

Reply:
I'm more concerned that we are repeatedly setting a value that EFA states is harmful for these p4/p5 instances that we are using in the examples 

> RDMAV_FORK_SAFE=1
> 
>Do not use. This is a RDMA-core environment variable. Prefer FI_EFA_FORK_SAFE (if it still makes sense for your Linux kernel version). The two look the same, but actually behaves very differently, especially on newer kernels, where RDMAV_FORK_SAFE=1 can break things.

> RDMAV_*
> 
> Do not use.

Reply:
reconciled with manifest linked by mhuguesaws, removed commented out fields and RMDAV_*